app/importers/est_data.rb

```diff
@@ -0,0 +1,62 @@
+class EstData
+  include Importer
+  LOGIN_URL = 'https://admin.export.gov/site_login'
+  ENDPOINT = 'https://admin.export.gov/admin/envirotech_solutions.json'
+
+  cattr_accessor :web_auth, :username, :password
+
+  COLUMN_HASH = {
+    'id'              => :source_id,
+    'name_chinese'    => :name_chinese,
+    'name_english'    => :name_english,
+    'name_french'     => :name_french,
+    'name_portuguese' => :name_portuguese,
+    'name_spanish'    => :name_spanish,
+    'created_at'      => :source_created_at,
+    'updated_at'      => :source_updated_at,
+  }.freeze
+
+  def initialize(resource = ENDPOINT)
+    @resource = resource
+  end
+
+  def import
+    data = JSON.parse(fetch_data)
+    articles = data.map { |article_hash| process_article_info article_hash }
+
+    Est.index articles
+  end
+
+  private
+
+  def fetch_data
+    headless_login
+    mechanize_agent.get(@resource).body
+  end
+
+  def headless_login
```

Comment:
i couldn't find a way to test this function in Rspec without the access tokens.. is there some way to do it?